app/src/main/java/com/duckduckgo/app/privacy/ui/PrivacyDashboardViewModel.kt

```diff
@@ -69,37 +69,37 @@ class PrivacyDashboardViewModel(
     init {
         pixel.fire(PRIVACY_DASHBOARD_OPENED)
         resetViewState()
-        domainsVisited.observeForever(domainsVisitedObserver)
-        trackerNetworkTally.observeForever(trackerNetworkActivityObserver)
+        sitesVisited.observeForever(sitesVisitedObserver)
+        trackerNetworkLeaderboard.observeForever(trackerNetworkActivityObserver)
     }
 
     @VisibleForTesting
     public override fun onCleared() {
         super.onCleared()
-        domainsVisited.removeObserver(domainsVisitedObserver)
-        trackerNetworkTally.removeObserver(trackerNetworkActivityObserver)
+        sitesVisited.removeObserver(sitesVisitedObserver)
+        trackerNetworkLeaderboard.removeObserver(trackerNetworkActivityObserver)
     }
 
-    fun onDomainsVisitedChanged(count: Int?) {
-        val domainCount = count ?: 0
-        val networkCount = viewState.value?.trackerNetworkTally?.count() ?: 0
+    fun onSitesVisitedChanged(count: Int?) {
+        val siteCount = count ?: 0
+        val networkCount = viewState.value?.trackerNetworkEntries?.count() ?: 0
         viewState.value = viewState.value?.copy(
-            showTrackerNetworkLeaderboard = showTrackerNetworkLeaderboard(domainCount, networkCount),
-            domainsVisited = domainCount
+            showTrackerNetworkLeaderboard = showTrackerNetworkLeaderboard(siteCount, networkCount),
+            sitesVisited = siteCount
         )
     }
 
-    fun onTrackerNetworkTallyChanged(tally: List<NetworkTally>?) {
-        val domainCount = viewState.value?.domainsVisited ?: 0
-        val networkTally = tally ?: emptyList()
+    fun onTrackerNetworkEntriesChanged(networkLeaderboardEntries: List<NetworkLeaderboardEntry>?) {
+        val domainCount = viewState.value?.sitesVisited ?: 0
+        val networkEntries = networkLeaderboardEntries ?: emptyList()
         viewState.value = viewState.value?.copy(
-            showTrackerNetworkLeaderboard = showTrackerNetworkLeaderboard(domainCount, networkTally.count()),
-            trackerNetworkTally = networkTally
+            showTrackerNetworkLeaderboard = showTrackerNetworkLeaderboard(domainCount, networkEntries.count()),
+            trackerNetworkEntries = networkEntries
         )
     }
 
-    private fun showTrackerNetworkLeaderboard(domainCount: Int, networkCount: Int): Boolean {
-        return domainCount > LEADERNOARD_MIN_DOMAINS_EXCLUSIVE && networkCount >= LEADERBOARD_MIN_NETWORKS
+    private fun showTrackerNetworkLeaderboard(siteVisitedCount: Int, networkCount: Int): Boolean {
+        return siteVisitedCount > LEADERBOARD_MIN_DOMAINS_EXCLUSIVE && networkCount >= LEADERBOARD_MIN_NETWORKS
```

Comment:
It adds complexity that one of these uses `>` and the other uses `>=`. Since they aren't used anywhere else, I'm guessing we can just use the same operator for both, and change the constant accordingly.

Again, I realise this code was already like that, though. Just highlighting now since my attention was drawn to it.

Reply:
It expresses the original spec which was that this would show:
-  after 30 sites
and
- when we have at least 3 networks

I think leaving it expresses that better.

Reply:
Then the spec isn't written perfectly clear. 

> 30
> 2

>=31
>=3

Either of these is preferable to mixing and matching. Anyway, not a suggestion that it needs immediately fixed, but we could make the code easier to read.

Reply:
I think it is readable in that we want 30 sites and there are 3 leaderboard slots in the UI. While it is generally nicer to be consistent switching to either >=31 or >2 just doesn't capture the way we would (and did) express this in natural language.

Deferring this discussion sounds good though, especially as the change wasn't introduced in this PR. Might poll the team on Monday to see if other people find it confusing too. Will be a good discussion.

Reply:
And while I'm at it... 

 `showTrackerNetworkLeaderboard` would be better named `shouldShowTrackerNetworkLeaderboard` 

Ok I'm done now

Reply:
That's not part of this PR but it's a good suggestion, I'll update.